src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocblocktaglocation/InputJavadocBlockTagLocationCustomTags.java

```diff
@@ -0,0 +1,16 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocblocktaglocation;
+
+/**
+ * configuration:
+ * tags="apiNote, implSpec, implNote"
+ */
+public class InputJavadocBlockTagLocationCustomTags {
+
+    /**
+     * Text. @apiNote note @implNote note @implSpec spec //warn
```

Comment:
please add standard tag to show that no violations are expected.
Please add few lines:
```
 text @apiNote note1 //warn
 text text @implNote note2  //warn
 text @implSpec  //warn
```

Reply:
> https://pbludov.github.io/issue-6929/sevntu-checkstyle/index.html#A1
> please send PR to fix javadoc

done: https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/774

Reply:
done

Reply:
> please add standard tag to show that no violations are expected.

not done.
Please add `text @since 1.0` to input.